concepts/50 React Components/40 Component Configuration Syntax/55 Markup Customization/0 Markup Customization.md

```diff
@@ -1,5 +1,7 @@
-Templates allow you to customize UI components. In the DevExtreme API, template properties end with `Template`: `itemTemplate`, `groupTemplate`, `contentTemplate`. When you specify them in React, replace `Template` in the name with `Render` or `Component`, depending on whether the template is a rendering function or custom component. For example, instead of `itemTemplate`, use `itemRender` or `itemComponent`. If a component does not reinitiate a template when the component rerenders, you should use the `itemComponent` property. Otherwise, we recommend that you use the `itemRender` property. 
+You can define custom content in DevExtreme components. We supply 2 types of properties for this task: ones that end with *'render'* (for example, [cellRender](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#cellRender)), and ones that end with *'component'* (for example, [cellComponent](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#cellComponent)).
```

Comment:
No big changes, just a number of minor comments. 

The word "define" seemed like it only described half of what we need to say. The ultimate goal is to present/display custom content. Maybe embed custom content. And it doesn't work well with the preposition "in". I used "display" and it feels more straightforward. 

"2" -> "two", because usually we spell out such numbers, unless it's a property value (or any other case where it's important to display the numeric form). 

Replaced "ones" with "those". And capitalized "render" and "component", since they seem to appear capitalized in property names. 

```suggestion
You can display custom content in DevExtreme components. We supply two types of properties for this task: those that end with *'Render'* (for example, [cellRender](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#cellRender)), and those that end with *'Component'* (for example, [cellComponent](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#cellComponent)).
```